bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/messages.properties

```diff
@@ -497,6 +497,11 @@ PreferencePageParameterValues_pageLabelSeparator = \ >\
 ThemingEnabled = E&nable theming
 ThemeChangeWarningText = Restart for the theme changes to take full effect
 ThemeChangeWarningTitle = Theme Changed
+RescaleAtRuntimeSettingChangeWarningTitle = DPI Setting Changed
+RescaleAtRuntimeSettingChangeWarningDesc = Restart for the effect to take place
+UpdateUIZoomGroupTitle = HiDPI settings
+UpdateUIZoomTitle = Monitor-specific UI scaling
+UpdateUIZoomDesc = EXPERIMENTAL! Activating this option will dynamically scale all windows according to the monitor they are currently in. It will also set the default browser to Edge in order to provide the appropriate scaling of content displayed in a browser. This feature is still in development and therefore considered experimental.
```

Comment:
Can we please improve the property names here?
E.g.:
- `UpdateUIZoomGroupTitle` -> `HiDpiSettingsGroupTitle`
- `UpdateUIZoomTitle` -> `RescaleAtRuntimeEnabled`
- `UpdateUIZoomTitle` -> `RescaleAtRuntimeDisclaimer`

And maybe add a mnemonic for the new option? If I see correctly "s" is not used yet, so you might use `Monitor-specific UI &scaling`